### DISCUSSION TITLE
Keep TLDr hidden or remove

### DISCUSSION THREAD

**Discussion by alwaisy** [dailydotdev/daily#1015] [Feature Requests]
I just noted Daily Dev has an AI TLDR feature. It is good to know about articles that make users curious to read or skip. But if it is kept hidden it looks better.

**Comment by idoshamun** [comment]
Hi @awaisalwaisy,

Can you please send a screenshot of the exact page or section you refer to? This will allow me to provide more context.

**Reply by alwaisy** [reply]
![image](https://github.com/dailydotdev/daily/assets/102944304/49e4cc8f-e0f4-4d09-9ecc-4ed9805c0720)


Removed TLDR:
![image](https://github.com/dailydotdev/daily/assets/102944304/33475eae-e237-4c8c-a3b1-fa79e482de2f)
A small toggle to show or hide TLDR. 


Why I close the discussion, I think it's one of the core features of daily.dev, if TLDR is hidden by default no clue for the user hwo about article. Title sometimes are not self explanatory.
